packages/demos/src/todomvc/webpack-config.js

```diff
@@ -1,46 +1,57 @@
-import CopyPlugin from 'copy-webpack-plugin';
-import {join} from 'path';
+// @ts-check
+const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
```

Comment:
Why did you remove the destructuring everywhere?

Reply:
Because I also use `resolve` here, and I like to have the `path` context when reading the call site. Therefore, I would suggest we never restructure from `path`. This is also already the case in some of the other files, e.g.`process-api-docs.js`.